src/mocks/EscapePodMock.ts

```diff
@@ -0,0 +1,10 @@
+import EscapePod from "../entities/EscapePod";
+
+
+const getUserMock = () => EscapePod.build({
```

Comment:
Seems either this `mocks/EscapePodMock.ts` or `mocks/EscapePod.ts` file should be deleted.